tests/test_integrated_channels/test_cornerstone/test_exporters/test_content_metadata.py

```diff
@@ -158,14 +159,29 @@ def test_transform_description(self, item_description, expected_description):
     )
     @responses.activate
     @ddt.unpack
-    def test_transform_key(self, item_key, expected_id):
+    def test_transform_key_1(self, item_key, expected_id):
         """
         Transforming a course key encodes the string if and only if invalid chars are present, otherwise it's a noop
         """
         item_content_metadata = merge_dicts(FAKE_SEARCH_ALL_COURSE_RESULT_3, item_key)
         exporter = CornerstoneContentMetadataExporter('fake-user', self.config)
         assert exporter.transform_course_key(item_content_metadata) == expected_id
 
+    @ddt.data(
+        (
+            {'key': 'testing+out+very+very+very+very+long+course+id+T2020'},
+        )
+    )
+    @responses.activate
+    @ddt.unpack
+    def test_transform_key_2(self, item_key):
```

Comment:
Think we do need to use ddt here since it's only one arg. Unless you plan to add more cases... it also makes test names harder to read in logs.